homeassistant/components/sensor/gtfs.py

```diff
@@ -38,15 +39,16 @@
     vol.Required(CONF_DESTINATION): cv.string,
     vol.Required(CONF_DATA): cv.string,
     vol.Optional(CONF_NAME): cv.string,
+    vol.Optional(CONF_OFFSET, default=datetime.timedelta(0)): cv.time_period_dict,
```

Comment:
About that, should I import `datetime.timedelta` at the top of the file, or go ahead and split this line?